etc/behat/services.xml

```diff
@@ -23,6 +23,9 @@
         <parameter key="sylius.behat.table_manipulator.class">Sylius\Behat\TableManipulator</parameter>
         <parameter key="sylius.behat.page.class">Sylius\Behat\Page\Page</parameter>
         <parameter key="sylius.behat.symfony_page.class">Sylius\Behat\Page\SymfonyPage</parameter>
+        <parameter key="sylius.behat.page.admin.crud.index.class">Sylius\Behat\Page\Admin\Crud\IndexPage</parameter>
+        <parameter key="sylius.behat.page.admin.crud.create.class">Sylius\Behat\Page\Admin\Crud\CreatePage</parameter>
+        <parameter key="sylius.behat.page.admin.crud.edit.class">Sylius\Behat\Page\Admin\Crud\EditPage</parameter>
```

Comment:
To be honest I found `EditPage` much more appropriate, but in terms of CRUD. On the other hand Resource action is named `Update`, so for now I would stay with **update**.